### PR TITLE
src/messages.c + include/messages.h

### PR DESCRIPTION
**Pull request by Fofifonfec**

### CODE REVIEW

File: intersynth_bluetooth/src/messages.c
Comment (Illugi317) [inline intersynth_bluetooth/src/messages.c:1]: do #include ../include/bluetooth.h instead. Not the actual source file.

**Review by Illugi317 — CHANGES_REQUESTED** [review]
Look at single comment regarding messages.c

**Comment by Fofifonfec** [comment]
ok @Illugi317  hvað um nú? Er þetta komið, eða kannski meira?

File: intersynth_bluetooth/src/messages.c
Comment (Illugi317) [reply intersynth_bluetooth/src/messages.c:1]: Define start of syssex and intersynth_id and also the end of the message thankoyu bye

**Comment by Illugi317** [comment]
Look at messages.c comments.

**Comment by Fofifonfec** [comment]
ok, define stuffs updated @Illugi317

**Review by Illugi317 — APPROVED** [review]
nice

**Pull request merged**